src/presets/ec/src/ec-default.scss

```diff
@@ -0,0 +1,141 @@
+@use "sass:map";
+@use '@ecl/theme-ec/index' as theme;
+
+@use "@ecl/vanilla-component-blockquote/blockquote-ec";
+@use "@ecl/vanilla-component-button/button-ec";
+@use "@ecl/vanilla-component-description-list/description-list";
+@use "@ecl/vanilla-component-link/link-ec";
+@use "@ecl/vanilla-component-ordered-list/ordered-list";
+@use "@ecl/vanilla-component-table/table";
+@use "@ecl/vanilla-component-unordered-list/unordered-list";
+@use "@ecl/vanilla-utility-typography/typography";
+
+// stylelint-disable no-descending-specificity
+.ecl button:not([class*='ecl-']),
+.ecl button:is([class*='ecl-u-']) {
+  @extend %ecl-button;
```

Comment:
I'm not really sure we need to creysate these placeholder selectors, i mean, their main idea is that they might be used to define some style that, if they are not extended, would not generate css, but here we are applying those selectors always in addition to some existing ones, so why not extending the one we have directly..?